config/deploy/production.rb

```diff
@@ -0,0 +1,48 @@
+# Simple Role Syntax
```

Comment:
If I were you I would remove all the cruft and just keep the server line on line 17.